src/store/login/saga.ts

```diff
@@ -148,4 +149,7 @@ export function* saga() {
     const action = yield take(SagaActionTypes.EmailLogin);
     success = yield call(emailLogin, action);
   } while (!success);
+
+  // After successful login
+  yield spawn(openFirstConversationAfterChannelsLoaded);
```

Comment:
updated to `spawn` after the `UserLogin` event. 

> Then, I'm guessing, we can probably remove the one from the registration flow since the event fires in both cases.

Little confused about this one. I see that we're firing the event in both cases (registration, login). But aren't these separate saga flows? i.e you're either registering (going through account setup) & then firing the event, or you're logging in, and then firing the event. Isn't that separate "events"? Or is there an overlap here?

Reply:
You're correct. We fire them via both flows. From an front end perspective what is a registration? It's effectively creating an account and then immediately logging in. The entire UI doesn't care if you got into the app through the login pages or the registration pages. All it cares about is that a new user is authenticated and we know who they are now. That's the UserLogin event. It can also actually happen from public zOS when you click the Connect button.